packages/lookit-api/src/types.ts

```diff
@@ -0,0 +1,76 @@
+export type Relationship = {
+  links: {
+    related: string;
+  };
+};
+
+export type Attributes = {
+  readonly pk?: number;
+  readonly url?: string;
+};
+
+export interface Relationships {}
```

Comment:
@okaycj could Relationships be something like this?
```ts
export interface Relationships = {
  links: Record<'related', string>
}
```

Reply:
Oh!  I am not sure, but I am going to try this.